docs/unit2.mdx

```diff
@@ -195,24 +198,22 @@ Call the first one `vim.txt` and the second one `nano.txt`.
 ### Deliverables
 
 **Required Tasks**
-
-- [ ]  Follow the directions above to set up your project directory. Take a screenshot and post it in the announcements thread.
-- [ ]  Complete the [Quiz](https://forms.gle/1whc3n6FjNNoq5ye9)
+ - Follow the directions above to set up your project directory. Take a screenshot and post it in the announcements thread.
+ - Complete the [Quiz](https://forms.gle/1whc3n6FjNNoq5ye9)
 
 **Optional** **Stretch Goals**
-
-- [ ]  Head to [Vim Adventures](https://vim-adventures.com/) and try to beat the puzzles! What object comes out of the final chest?
-- [ ]  Add some custom code to your .bashrc or .zshrc
+ - Head to [Vim Adventures](https://vim-adventures.com/) and try to beat the puzzles! What object comes out of the final chest?
+ - Add some custom code to your .bashrc or .zshrc
 
 ### Due Date
 
-Monday, January 17 by 11:59pm
+Monday, April 11 by 11:59pm
 
 ### Submission
 
 Send your screenshot in the Discord channel and complete [the quiz](https://forms.gle/huFVCdN48dcfty256) for this week.
 
-<iframe src="https://docs.google.com/forms/d/e/1FAIpQLSdn3Mrv8_RShKjSLfD-th7L0LrNWVyXjtfEuQzz_EWMFhwGHQ/viewform?embedded=true" width="640" height="1056" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
+<iframe src="https://docs.google.com/forms/d/e/1FAIpQLSdn3Mrv8_RShKjSLfD-th7L0LrNWVyXjtfEuQzz_EWMFhwGHQ/viewform?embedded=true" width="100%" height="1500" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
```

Comment:
Is this height good? I picked 1500 for unit 1 because that's roughly what the height came out to based on the number of questions. Make sure this also doesn't add extra white space to the bottom.

Reply:
I fixed the height. There is no scrollbar, and there is a just a little bit of whitespace for better spacing.